modules/javafx.graphics/src/main/java/javafx/application/ConditionalFeature.java

```diff
@@ -150,6 +152,15 @@ public enum ConditionalFeature {
      */
     UNIFIED_WINDOW,
 
+    /**
+     * Indicates that a system supports {@link StageStyle#EXTENDED} and {@link StageStyle#EXTENDED_UTILITY}.
+     * <p>
+     * This feature is currently supported on Windows, Linux, and macOS.
+     *
+     * @since 25
+     */
+    EXTENDED_WINDOW,
+
```

Comment:
Is this PR blocked by the Preview Feature Support #1359?

Reply:
Yes, it is. Once we get preview features in, I'll annotate the new API elements.